<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1511</width>
    <height>961</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>-1</y>
     <width>1521</width>
     <height>971</height>
    </rect>
   </property>
   <property name="maximumSize">
    <size>
     <width>1521</width>
     <height>971</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">
color: rgb(14, 70, 255);
alternate-background-color: rgb(208, 196, 255);
color: rgb(6, 6, 6);
font: 13pt &quot;MS Shell Dlg 2&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="widget">
    <attribute name="title">
     <string>Malware Detect</string>
    </attribute>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>-1</y>
       <width>1521</width>
       <height>951</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>pic.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>240</y>
       <width>1161</width>
       <height>611</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color:rgb(255, 255, 255,200)</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>490</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(184, 133, 255);
color: rgb(255, 255, 255);
font: 18pt &quot;MS Shell Dlg 2&quot;;
 box-shadow: 0 5px #666;
 border-radius: 10px;
 transition-duration: 0.4s;
display: inline-block;
cursor: pointer;</string>
     </property>
     <property name="text">
      <string>Scan file</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_2">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>670</y>
       <width>561</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(10, 10, 10);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>730</y>
       <width>291</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 18pt &quot;MS Shell Dlg 2&quot;;
color: rgb(255, 255, 255);
background-color: rgb(183, 149, 255);
 box-shadow: 0 5px #666;
 border-radius: 10px;
 transition-duration: 0.4s;
display: inline-block;
cursor: pointer;</string>
     </property>
     <property name="text">
      <string>Check Malware</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>590</y>
       <width>991</width>
       <height>91</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(10, 10, 10);</string>
     </property>
     <property name="text">
      <string>Enter a link </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>710</x>
       <y>400</y>
       <width>851</width>
       <height>101</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 20pt &quot;MS Shell Dlg 2&quot;;
color: rgb(8, 8, 8);</string>
     </property>
     <property name="text">
      <string>Select a file </string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>Phishing Email</string>
    </attribute>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>-20</x>
       <y>0</y>
       <width>1541</width>
       <height>951</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>phishing_background.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>274</x>
       <y>319</y>
       <width>901</width>
       <height>261</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,200);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>580</x>
       <y>380</y>
       <width>481</width>
       <height>51</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(3, 3, 3);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Email Phishing checker</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_4">
     <property name="geometry">
      <rect>
       <x>530</x>
       <y>450</y>
       <width>431</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(4, 4, 4);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_5">
     <property name="geometry">
      <rect>
       <x>650</x>
       <y>530</y>
       <width>201</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(7, 7, 7);
background-color: rgb(255, 92, 95);
font: 14pt &quot;MS Shell Dlg 2&quot;;
 box-shadow: 0 5px #666;
 border-radius: 10px;
 transition-duration: 0.4s;
display: inline-block;
cursor: pointer;</string>
     </property>
     <property name="text">
      <string>Check Email</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_3">
    <attribute name="title">
     <string>Hashing Text</string>
    </attribute>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>-6</x>
       <y>-1</y>
       <width>1521</width>
       <height>951</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>hashing_background.jpg</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>420</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 246, 253);
 box-shadow: 0 5px #666;
 border-radius: 10px;
 transition-duration: 0.4s;
display: inline-block;
cursor: pointer;</string>
     </property>
     <property name="text">
      <string>MD5</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>200</y>
       <width>471</width>
       <height>191</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(7, 7, 7);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>210</x>
       <y>110</y>
       <width>601</width>
       <height>131</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>Enter a text </string>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>520</y>
       <width>751</width>
       <height>61</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(255, 255, 255);
font: 20pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
     <property name="text">
      <string>The hash text</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_4">
     <property name="geometry">
      <rect>
       <x>340</x>
       <y>420</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 16pt &quot;MS Shell Dlg 2&quot;;
background-color: rgb(255, 246, 253);
 box-shadow: 0 5px #666;
 border-radius: 10px;
 transition-duration: 0.4s;
display: inline-block;
cursor: pointer;</string>
     </property>
     <property name="text">
      <string>SHA1</string>
     </property>
    </widget>
    <widget class="QPlainTextEdit" name="plainTextEdit_3">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>580</y>
       <width>461</width>
       <height>231</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255,180);
color: rgb(7, 7, 7);
font: 16pt &quot;MS Shell Dlg 2&quot;;</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
